### ISSUE TITLE
Where's the source?

### ISSUE THREAD

**Issue by matthewp** [bahmutov/babel-service#6]
Sorry if I just overlooked it, but it seems like there's several seperate things bundled into babel-service.js, including the service worker code. Should that be somewhere in the project?

**Comment by bahmutov** [comment]
Yeah, I did not have time to create a build step, so I put feature tests, my code and babel into babel-service.js file. I will split the source into dependencies and will add a build step.

**Comment by matthewp** [comment]
Great, thank you.
